tests/fixtures/generic/ini_nested__plasmarc.json

```diff
@@ -0,0 +1,280 @@
+      "wallpaperplugin": "org.kde.image",
+      "Applets": [
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.kickoff",
+          "Configuration": {
+            "PreloadWeight": "100",
+            "popupHeight": "514",
+            "popupWidth": "645",
+            "ConfigDialog": {
+              "DialogHeight": "539",
+              "DialogWidth": "719"
+            },
+            "General": {
+              "favoritesPortedToKAstats": "true"
+            },
+            "Shortcuts": {
+              "global": "Alt+F1"
+            }
+          },
+          "Shortcuts": {
+            "global": "Alt+F1"
+          }
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.pager"
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.icontasks",
+          "Configuration": {
+            "General": {
+              "launchers": "applications:systemsettings.desktop,applications:org.kde.discover.desktop,preferred://filemanager,preferred://browser,applications:com.discordapp.Discord.desktop,applications:sublime_text.desktop,applications:sublime_merge.desktop,applications:org.kde.konsole.desktop"
+            }
+          }
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.marginsseparator"
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.systemtray",
+          "Configuration": {
+            "PreloadWeight": "100",
+            "SystrayContainmentId": "8"
+          }
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.digitalclock",
+          "Configuration": {
+            "PreloadWeight": "100",
+            "popupHeight": "450",
+            "popupWidth": "396",
+            "Appearance": {
+              "dateFormat": "isoDate"
+            },
+            "ConfigDialog": {
+              "DialogHeight": "539",
+              "DialogWidth": "719"
+            }
+          }
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.showdesktop"
+        },
+        {
+          "immutability": "1",
+          "plugin": "org.kde.plasma.colorpicker",
+          "Configuration": {
+            "PreloadWeight": "60",
+            "General": {
+              "history": "#001e08,#001d06,#ff0000,#ffff00,#0080fe,#f30101,#01f301,#ff8040"
+            }
+          }
+        }
+      ],
+      "ConfigDialog": {
+        "DialogHeight": "84",
+        "DialogWidth": "1707"
+      },
+      "General": {
+        "AppletOrder": "3;4;5;6;27;7;18;19"
+      }
+    },
+    {
+      "activityId": "",
+      "formfactor": "2",
+      "immutability": "1",
+      "lastScreen": "0",
+      "location": "4",
+      "plugin": "org.kde.plasma.private.systemtray",
+      "popupHeight": "431",
+      "popupWidth": "431",
+      "wallpaperplugin": "org.kde.image",
+      "Applets": [
```

Comment:
It seems like information is being lost in the conversion. Should there be some sort of `_item` key to provide the original index number?